information-mediator/docs/main.md

```diff
@@ -0,0 +1,473 @@
+``` bash
+ $ helm upgrade --install information-mediator ./information-mediator/ -f val.yaml 
+```
+
+11. Structure of Helm charts
+
+The "information-mediator" chart contains the following sub-charts:
+
+* A chart for [X-Road Central Server](information_mediator/charts/xroad-cs)
+  the sub-chart contains also the following templates:
+    - [application.yaml](information_mediator/charts/xroad-cs/templates/application.yaml) for installing X-Road Central Server pod and service to Kubernetes Cluster
+    - [postgres.yaml](information_mediator/charts/xroad-cs/templates/postgres.yaml) for installing X-Road Central Server PostgresSQL database to Kubernetes Cluster
+    - [secret.yaml](information_mediator/charts/xroad-cs/templates/secret.yaml) for installing X-Road Central Server secrets to Kubernetes Cluster
+* A chart for [X-Road management services Security Server](information_mediator/charts/xroad-ssm)
+  the sub-chart contains also the following templates:
+    - [application.yaml](information_mediator/charts/xroad-ssm/templates/application.yaml) for installing X-Road management services Security Server pod and service to Kubernetes Cluster
+    - [postgres.yaml](information_mediator/charts/xroad-ssm/templates/postgres.yaml) for installing X-Road management services Security Server PostgresSQL database to Kubernetes Cluster
+    - [secret.yaml](information_mediator/charts/xroad-ssm/templates/secret.yaml) for installing X-Road management services Security Server secrets to Kubernetes Cluster
+* A chart for [X-Road consumer Security Server](information_mediator/charts/xroad-ssc)
+  the sub-chart contains also the following templates:
+    - [application.yaml](information_mediator/charts/xroad-ssc/templates/application.yaml) for installing X-Road consumer Security Server pod and service to Kubernetes Cluster
+    - [postgres.yaml](information_mediator/charts/xroad-ssc/templates/postgres.yaml) for installing X-Road consumer Security Server PostgresSQL database to Kubernetes Cluster
+    - [secret.yaml](information_mediator/charts/xroad-ssc/templates/secret.yaml) for installing X-Road comnsumer Security Server secrets to Kubernetes Cluster
+* A chart for [X-Road provider Security Server](information_mediator/charts/xroad-ssp)
+  the sub-chart contains also the following templates:
+    - [application.yaml](information_mediator/charts/xroad-ssp/templates/application.yaml) for installing X-Road provider Security Server pod and service to Kubernetes Cluster
+    - [postgres.yaml](information_mediator/charts/xroad-ssp/templates/postgres.yaml) for installing X-Road provider Security Server PostgresSQL database to Kubernetes Cluster
+    - [secret.yaml](information_mediator/charts/xroad-ssp/templates/secret.yaml) for installing X-Road provider Security Server secrets to Kubernetes Cluster
+
+12. CI / CD pipeline
+
+There is also a [CircleCI pipeline](.circleci/config.yml) created for automation of those previous steps.
```

Comment:
These links do not work anymore as you have moved the documentation.

Reply:
Fix c84a4f112a5053772e60042ca260d90705e09962